include/c64_screenmap.h

```diff
@@ -0,0 +1,275 @@
+/*****************************************************************************/
+/*                                                                           */
+/* c64_screenmap.h                                                           */
+/*                                                                           */
+/* (c) Copyright 2019, Gerhard W. Gruber (sparhawk@gmx.at)                   */
+/*                                                                           */
+/* When using C64 mode, this include converts the characters                 */
+/* from PETSCII to screen mapping, so you can write directly to              */
+/* the screen memory.                                                        */
+/*                                                                           */
+/* If this include is used, no additional macros are needed                  */
+/*                                                                           */
+/*****************************************************************************/
+
+#ifndef _C64_SCREENMAP_H
+#define _C64_SCREENMAP_H
+
+#pragma charmap ($02, $82)
+#pragma charmap ($01, $81)
+#pragma charmap ($00, $80)
```

Comment:
Why is the order $02, $01, $00 ?

Reply:
I think this probalby happend when I copy&pasted it. No particular reason. I can change it if required.